src/background/popupHandler.ts

```diff
@@ -17,7 +17,7 @@ export const getAeppUrl = (v: any) => new URL(v.connection.port.sender.url);
 export const showPopup = async (aepp: any, type: string, params?: any) => {
   const id = uuid();
   const { href, protocol, host } = typeof aepp === 'object' ? getAeppUrl(aepp) : new URL(aepp);
-  const tabs = await browser.tabs.query({ active: true });
+  const tabs: any[] = await browser.tabs.query({ active: true });
```

Comment:
I don't see a reason for this change. The type is inherited from the `Browser` interface.